src/OpenTelemetry.ClrProfiler.Native/logger.h

```diff
@@ -16,7 +16,7 @@ struct TracerLoggerPolicy
 #ifdef _WIN32
     // this field will be removed once merged with the profiler in order to have
     // the same product folder name
-    inline static const WSTRING folder_path = WStr(R"(Datadog .NET Tracer\logs)");
+    inline static const WSTRING folder_path = WStr(R"(OpenTelemetry .NET AutoInstrumentation\logs)");
```

Comment:
Not for now but we may want to review this folder name is really long ... I guess I suggested it in the past but I changed my mind 😬